<?xml version="1.0"?>
<!--***********************************************************
	Risky Mitigating Policy Data Reader
	Emily Sheetz, NSTGRO VTE 2024

	run either:
		$ roslaunch safety_aware_reasoning risk_mitigating_policy_data_reader.launch
		OR
		$ rosrun safety_aware_reasoning risk_mitigating_policy_data_reader.py
************************************************************-->
<launch>
	<arg name="robot" default="val"/>
	<arg name="env" default="lunar_habitat"/>
	<arg name="human_generated_data" default="true"/>

	<node pkg="safety_aware_reasoning" type="risk_mitigating_policy_data_reader.py" name="RiskMitigatingPolicyDataReader" output="screen">
		<param name="robot" value="$(arg robot)"/>
		<param name="environment" value="$(arg env)"/>
		<param name="human_generated_data" value="$(arg human_generated_data)"/>
	</node>
</launch>
